internal/action/show.go

```diff
@@ -195,8 +196,8 @@ func (s *Action) showHandleOutput(ctx context.Context, name string, sec gopass.S
 		}
 	}
 
-	if IsClip(ctx) && pw != "" {
-		if err := clipboard.CopyTo(ctx, name, []byte(pw), s.cfg.ClipTimeout); err != nil {
+	if (IsClip(ctx) || config.Bool(ctx, "core.showautoclip")) && pw != "" {
```

Comment:
Same here, the config is `core.autoclip` but we're using `showautoclip` instead
```suggestion
	if (IsClip(ctx) || config.Bool(ctx, "core.autoclip")) && pw != "" {
```

Reply:
Actually it seems the name change is on purpose, but the migration path doesn't migrate correctly?

Reply:
This is actually intentional. We have `core.autoclip` for generate and `core.showautoclip` for gopass show.